bin/version

```diff
@@ -34,7 +34,7 @@ DUMMY_FORMATS = [
 
 def get_versions(builddir):
     """Yield paths to each documented version."""
-    yield from pathlib.Path(builddir).glob('[0-9]*.*')
+    yield from pathlib.Path(builddir).glob('[0-9]*.[0-9]*')
```

Comment:
Haven't manually tested this. Will this yield the alphas/betas correctly?

Reply:
Seeing as we have the convention of always specifying the minor/2nd number, this should do it. Will check with Oliver

Reply:
Think that's ok.